lib/tasks/book2.rake

```diff
@@ -32,7 +32,21 @@ end
 def genbook(code, &get_content)
   template_dir = File.join(Rails.root, "templates")
 
-  nav = '<div id="nav"><a href="[[nav-prev]]">prev</a> | <a href="[[nav-next]]">next</a></div>'
+  nav = '
```

Comment:
Oops, I didn't mean for this file to be in this PR, but it'll soon be in #1179 anyhow.